### ISSUE TITLE
Verbose mode is not printing the istanbul config

### ISSUE THREAD

**Issue by bennyn** [mattlewis92/karma-coverage-istanbul-reporter#43]
When using `istanbul report --root ./coverage --dir ./coverage/report --verbose html`, then [istanbul](https://github.com/istanbuljs/istanbuljs) prints the applied istanbul configuration. 

**Example:**

```yml
Using configuration
-------------------
verbose: true
instrumentation:
    root: .
    extensions:
        - .js
    default-excludes: true
    excludes:
        - '**/*test.*.js'
    variable: __coverage__
    compact: true
    preserve-comments: false
    complete-copy: false
    save-baseline: false
    baseline-file: ./coverage/coverage-baseline.raw.json
    include-all-sources: false
    include-pid: false
    es-modules: false
    auto-wrap: false
    ignore-class-methods: []
reporting:
    print: detail
    reports:
        - html
    dir: ./coverage/node
    summarizer: pkg
    report-config: {}
    watermarks:
        statements: [50, 80]
        functions: [50, 80]
        branches: [50, 80]
        lines: [50, 80]
hooks:
    hook-run-in-context: false
    hook-run-in-this-context: false
    post-require-hook: null
    handle-sigint: false
check:
    global:
        statements: 100
        lines: 100
        branches: 100
        functions: 100
        excludes: []
    each:
        statements: 0
        lines: 0
        branches: 0
        functions: 0
        excludes: []

-------------------
```

I wanted to achieve the same with "karma-coverage-istanbul-reporter", so I configured "verbose" in my `karma.conf.js` file:

```js
coverageIstanbulReporter: {
  reports: ['html', 'json', 'text'],
  dir: path.join(__dirname, 'coverage', 'browser'),
  combineBrowserReports: true,
  fixWebpackSourcePaths: true,
  'report-config': {
    html: {
      subdir: 'html',
      verbose: true
    }
  }
}
```

Unfortunately, it doesn't print the istanbul configuration. My guess is that the "verbose" flag is not properly forwarded to istanbul.

I am using **karma-coverage-istanbul-reporter v1.4.2** and **istanbul v1.1.0-alpha.1**.

**Comment by mattlewis92** [comment]
Thanks for reporting, this will be fixed in `1.4.3` (FYI `verbose` needs to be passed as a top level property to `coverageIstanbulReporter` to get this to work)

**Comment by bennyn** [comment]
I just tested with "karma-coverage-istanbul-reporter" v2.0.0 and there it works. Thanks for acting that quick! 👍
